.travis.yml

```diff
@@ -15,6 +16,8 @@ env:
   - TEST_SUITE=vmdb PARALLEL=true
   - TEST_SUITE=brakeman
 matrix:
+  allow_failures:
+  - rvm: 2.5.3
```

Comment:
You can't have allow_failures as a key twice like this...I think you need to do

```yaml
matrix:
  allow_failures:
  - rvm: 2.5.3
  - rvm: 2.4.2
```

However on that note, when Ruby was bumped to 2.4.5, the allow_failures for 2.4.2 no longer made sense.  @jrafanie I'm guessing you just missed this?  We can probably drop 2.4.2 allow_failures, IMO.

Reply:
@Fryguy ok, updated.